libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -26,20 +26,26 @@ import pubsub,
 logScope:
   topics = "gossipsub"
 
-const GossipSubCodec* = "/meshsub/1.0.0"
+const
```

Comment:
can we please keep the style consistent across the board? The current style is to use a declaration per line ie: 

```nim
const MyConst1
const MyConst2
``` 
vs

```nim
const
  MyConst1
  MyConst2
```

Please continue using the former.

Reply:
We should probably favour the latter style for all declarations, to avoid repeating the same keyword.

Reply:
is there a clang-format equivalent that works for nim? I think there was something similar but not sure would be so comprehensive.
Everyone has it's own style and ways of typing.. a common formatting scheme that automatically gets applied is the only thing that would leave us without any stress...

Reply:
Why? It is purely a style issue and both are valid, personal preferences are irrelevant here as this is a subjective issue. I don't want to rewrite the entire code base just to change it to the latter and I don't want the codebase to end up with two competing styles that will only confuse future contributors.

Reply:
> is there a clang-format equivalent that works for nim? I think there was something similar but not sure would be so comprehensive.
> Everyone has it's own style and ways of typing.. a common formatting scheme that automatically gets applied is the only thing that would leave us without any stress...

Totally agree, the last thing we want is a tabs vs spaces argument :), this things should be taken care of by a formatter.

Reply:
there is `nimpretty`... will try but last time I tried barely did indents

Reply:
yeah It did not cover this case...

Reply:
fyi, it's indeed just indent and line length fix.. not really a formatter

Reply:
fwiw, most of the other (nbc/nimbus) code uses the multiple-consts-per-`const` style

Reply:
same with `type` where there's actually a reason: within a type block, order doesn't matter

Reply:
> fwiw, most of the other (nbc/nimbus) code uses the multiple-consts-per-`const` style

Sure, I'm definitely not arguing for which style is better, I'm arguing for consistency. This codebase, for whatever reason, ended up using the former and to keep it consistent we should stick to it or refactor. But, I'd hate to see two or more styles proliferate for no good reason.

Reply:
btw, whatever style is more idiomatic Nim is the one we should stick to, but I've seen both in the wild, so not sure if there is such a thing.

Reply:
> same with `type` where there's actually a reason: within a type block, order doesn't matter

`Type` is being used in a multiline manner, pretty much everywhere across the codebase (afaik).